resources/assets/coffee/react/profile-page/links.coffee

```diff
@@ -103,6 +103,14 @@ export class Links extends React.PureComponent
       for row, j in rows when _.compact(row).length > 0
         div key: j, className: "#{bn}__row #{bn}__row--#{j}", row
 
+      div
+        className: 'edit-profile-links'
+        a
+          href: laroute.route('account.edit')
+          button
```

Comment:
`button` can't be inside `a`. I don't see the point anyway (apart of skipping some styling).